.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+      - name: "Create .zip filename"
+        id: zipfn
+        shell: bash
+        run: echo "FILEPATH=build/${{ steps.basefn.outputs.FILEPATH }}.zip" >> $GITHUB_OUTPUT
+      - name: "Get extension"
+        id: ext
+        shell: bash
+        run: |
+          if [ ${{ runner.os }} == 'Windows' ]; then
+            echo "EXT=.exe" >> $GITHUB_OUTPUT
+          elif [ ${{ runner.os }} == 'Linux' ]; then
+            echo "EXT=" >> $GITHUB_OUTPUT
+          else
+            echo "EXT=.dmg" >> $GITHUB_OUTPUT
+          fi
+      - name: "Get jpackage output filepath"
+        id: jpackagefn
+        shell: bash
+        run: |
+          # TODO Issue-662 sync version number with Main.java and build.gradle
+          if [ ${{ runner.os }} == 'Windows' ]; then
+            echo "FILEPATH=build/jpackage/RCTab-1.3.1.exe" >> $GITHUB_OUTPUT
+          elif [ ${{ runner.os }} == 'Linux' ]; then
+            echo "FILEPATH=build/jpackage/rctab_1.3.1-1_amd64.deb" >> $GITHUB_OUTPUT
+          else
+            echo "FILEPATH=build/jpackage/RCTab-1.3.1.dmg" >> $GITHUB_OUTPUT
+          fi
+      - name: "Create executable filename"
+        id: exefn
+        shell: bash
+        run: echo "FILEPATH=build/${{ steps.basefn.outputs.FILEPATH }}${{ steps.ext.outputs.EXT }}" >> $GITHUB_OUTPUT
+          
+      - uses: actions/checkout@v3
+
+      - name: "Set up JDK 17.0.2"
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17.0.2'
+          distribution: 'temurin'
+
+      - name: "Validate Gradle wrapper"
+        uses: gradle/wrapper-validation-action@ccb4328a959376b642e027874838f60f8e596de3
+
+
+      - name: "Create zip with jlinkZip"
+        uses: ./.github/actions/gradle-and-sha
+        with:
+          gradle-command: jlinkZip
+          intermediate-filepath: build/rcv.zip
+          final-filepath: ${{ steps.zipfn.outputs.FILEPATH }}
+
+      - name: "Create executable with jpackage"
+        uses: ./.github/actions/gradle-and-sha
+        with:
+          gradle-command: jpackage
+          intermediate-filepath: ${{ steps.jpackagefn.outputs.FILEPATH }}
+          final-filepath: ${{ steps.exefn.outputs.FILEPATH }}
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: Package
+          if-no-files-found: error
+          path: |
+            ${{ github.workspace }}/${{ steps.zipfn.outputs.FILEPATH }}
+            ${{ github.workspace }}/${{ steps.zipfn.outputs.FILEPATH }}.sha512
+            ${{ github.workspace }}/${{ steps.exefn.outputs.FILEPATH }}
+            ${{ github.workspace }}/${{ steps.exefn.outputs.FILEPATH }}.sha512
+          retention-days: 90
+
+      - name: "Upload binaries to release"
+        uses: svenstaro/upload-release-action@v2
+        if: ${{ github.event_name }} == 'release'
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: ${{ github.workspace }}/${{ steps.zipfn.outputs.FILEPATH }}*
```

Comment:
Is it only uploading the jlinkzip version since this is zipfn here? I think we'd also want to upload the jpackage versions.

I also noticed that the artifact file you linked to for Linux testing was a single file with no extension that turned out to be a zip with every single platform and SHA in it. Hopefully we're not uploading something like that to the release, as it was fairly confusing. I'd prefer individual files as the current release.yml does.

Reply:
1. Before we merge this, we need to decide whether we want to replace jlinkzip with jpackage, release both, or continue just releasing jlinkzip until we've had more time/feedback on jpackage. I believe that decision is [pending feedback from vendors](https://github.com/BrightSpots/rcv/pull/660#issuecomment-1523536314).
2. Good catch on the linux release! No, that's not intentional. I'll fix it.

Reply:
Oh, and I didn't notice the RCVRC logo at any point. The installer just had a red CD-looking thing, and the program icon was the Java Star Trek looking icon.

Reply:
Ah, I understand: yes, that is a limitation of Github Artifacts on Actions. In a release, they will be individual files, but Github forces them all into one giant package on Github Actions Artifacts.

i.e. this won't be an issue when we create releases, only in development.